SwrveSDK/src/main/java/com/swrve/sdk/SwrveWakefulService.java

```diff
@@ -24,9 +24,14 @@ public SwrveWakefulService() {
     @Override
     protected void onHandleIntent(Intent intent) {
         try {
-            if (hasValidExtra(intent)) {
-                handleSendEvents(intent);
-            } else {
+            boolean unkownIntentContent = true;
+            ArrayList<String> eventsExtras = intent.getExtras().getStringArrayList(EXTRA_EVENTS);
+            if (eventsExtras != null) {
+                unkownIntentContent = false;
+                handleSendEvents(eventsExtras);
+            }
```

Comment:
just add an else block that prints the error, and remove the boolean unkownIntentContent.